result/isislsp.yaml

```diff
@@ -0,0 +1,356 @@
+components:
+  schemas:
+    IsisLsps.State.Request:
+      description: >- 
+        The request to retrieve ISIS Link State Protocol Data Unit (LSP) information learned by this router.
```

Comment:
- Should be "ISIS Link State PDUs (LSP)" - for same reason why we didn't expand ISIS
- Correction - "information learned by the routers"